### PR TITLE
Fixing test case for get_location_string method.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR fixes the test case made it for get_location_string method in favor of this [OSPR](https://github.com/eduNEXT/flow-control-xblock/pull/25).

## Test:

 - Activate the virtual environment python.
 - Run the **tox** command.

## Reviewers:

- [ ] @felipemontoya 
- [ ] @diegomillan

### CODE REVIEW

File: flow_control/tests/test_flowcontrol.py [hunk 1]
```diff
@@ -98,29 +99,38 @@ def test_get_location_string(self):
...
-            type=self.block.course_id.BLOCK_TYPE_PREFIX,
-            type_id=resource,
-            locator=locator)
-
-        self.assertEqual(testing_string, result_string)
-        self.block.course_id.to_deprecated_string.assert_called_with()
+        # We want to pass arbitrary times for random.choice.
+        for index in range(5):  # pylint: disable=unused-variable
+            self.block.course_id = MagicMock()
+            self.block.course_id.BLOCK_PREFIX = random.choice(course_strings)
```
Comment (felipemontoya) [inline flow_control/tests/test_flowcontrol.py]: Why are you making this random? is it not enough to go through all the course_strings one by one?

File: flow_control/tests/test_flowcontrol.py [hunk 2]
```diff
@@ -98,29 +99,38 @@ def test_get_location_string(self):
         It should return the problem location string given its Id
         """
 
+        course_strings = [
+            'course',
+            'course-v1',
+            'Course',
+            'course+course',
```
Comment (felipemontoya) [inline flow_control/tests/test_flowcontrol.py]: Do you think this should be a valid string? or are you expecting the tests to fail when they reach this value?

File: flow_control/tests/test_flowcontrol.py (hunk 1 above)
Comment (Squirrel18) [reply flow_control/tests/test_flowcontrol.py]: This random is just for get different strings for BLOCK_PREFIX and BLOCK_TYPE_PREFIX. 
Could be made it in other way? @felipemontoya

File: flow_control/tests/test_flowcontrol.py (hunk 2 above)
Comment (Squirrel18) [reply flow_control/tests/test_flowcontrol.py]: It is just for fail in case. @felipemontoya

**Comment by Squirrel18** [comment]
One more review please. @felipemontoya @diegomillan

**Review by felipemontoya — APPROVED** [review]

**Review by diegomillan — APPROVED** [review]

**Comment by Squirrel18** [comment]
@felipemontoya @diegomillan Do you think it is ready to be merged?

**Pull request merged**